src/customers/customer-account-create-order.md

```diff
@@ -60,6 +70,14 @@ If your store has multiple views, choose the store view where the order is to be
 
    - Click <span class="btn">Add Selected Product(s) to Order</span> to update the cart.
 
+1. Configure the product [gift options]({% link sales/gift-options.md %}):
+
+   - Click **Gift Options**.
+
+   - Complete the `Gift Message` fields or select the `Gift Wrapping`.
+
+   - Click <span class="btn">OK</span>.
```

Comment:
I think it is simpler to just link to this since it depends on the product configuration.

```suggestion
1. If a product is configured for [gift options]({% link sales/gift-options.md %}), set the options as needed.
```